rust/gitxetcore/src/config/xet.rs

```diff
@@ -678,6 +682,24 @@ mod config_create_tests {
         assert_err!(load_profile(&cfg, None, &repo_info));
     }
 
+    #[test]
+    fn test_load_profile_succeed_multiple_identical_profiles() {
+        let mut cfg = Cfg::with_default_values();
+        let dev_profile = get_test_dev_profile();
+        let profiles = cfg.profiles.as_mut().unwrap();
+        profiles.insert("prod".to_string(), dev_profile);
+        let repo_info = RepoInfo {
+            env: XetEnv::Custom,
+            remote_urls: vec![
+                "https://xethubdev.com/org/repo".to_string(),
+                "https://xethubdev.com/org/repo".to_string(),
```

Comment:
nit: we should probably also test that a "fork" is properly deduped (e.g. `https://xethubdev.com/user/repo`)

Reply:
done